rialto_airflow/dags/harvest.py

```diff
@@ -171,14 +171,21 @@ def link_funders(snapshot, openalex_fill_in, dimensions_fill_in, wos_fill_in):
         return count
 
     @task()
-    def publish_openaccess(snapshot):
+    def publish_open_access(snapshot):
         openaccess.write_publications(snapshot)
         openaccess.write_contributions(snapshot)
         openaccess.write_contributions_by_school(snapshot)
         openaccess.write_contributions_by_department(snapshot)
 
     @task()
-    def upload_publish_files(snapshot):
+    def publish_data_quality(snapshot):
+        data_quality.write_authors(snapshot)
+        data_quality.write_sulpub(snapshot)
+        data_quality.write_contributions_by_source(snapshot)
+        data_quality.write_publications(snapshot)
+
```

Comment:
I renamed this to distinguish it from the `upload_data_quality_files()`. Perhaps it would simplify things to do them all at once in a separate DAG but I didn't take that step yet.